### ISSUE TITLE
Crash report 5.6: HelpShift crashes, NPE in  com.helpshift.activities.MainActivity.onCreate

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#4450]
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/com.helpshift.support.activities.ParentActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context android.support.v4.app.FragmentHostCallback.getContext()' on a null object reference
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3254)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3350)
       at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5387)
       at android.app.ActivityThread.access$1200(ActivityThread.java:222)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1801)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:158)
       at android.app.ActivityThread.main(ActivityThread.java:7229)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context android.support.v4.app.FragmentHostCallback.getContext()' on a null object reference
       at android.support.v4.app.FragmentState.instantiate(Fragment.java:100)
       at android.support.v4.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:2047)
       at android.support.v4.app.Fragment.restoreChildFragmentState(Fragment.java:1278)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1053)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1286)
       at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1268)
       at android.support.v4.app.FragmentManagerImpl.dispatchCreate(FragmentManager.java:2138)
       at android.support.v4.app.FragmentController.dispatchCreate(FragmentController.java:190)
       at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:349)
       at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:85)
       at com.helpshift.activities.MainActivity.onCreate(MainActivity.java:13)
       at com.helpshift.support.activities.ParentActivity.onCreate(ParentActivity.java:21)
       at android.app.Activity.performCreate(Activity.java:6876)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1135)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3207)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3350)
       at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5387)
       at android.app.ActivityThread.access$1200(ActivityThread.java:222)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1801)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:158)
       at android.app.ActivityThread.main(ActivityThread.java:7229)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by maxme** [comment]
It's probably a support library issue introduced in 24.0.0 (and apparently not fixed in latest 24.1.1) https://code.google.com/p/android/issues/detail?id=208911

**Comment by maxme** [comment]
Confirmed, it's a support lib crash, moving this to 5.7.

**Comment by hypest** [comment]
### Steps to reproduce:

* Open the app and visit the /me screen
* Go to "Help & Support"
* Tap on "Contact us"
* While on the "Support chat" screen, rotate the device
* 💥

**Comment by hypest** [comment]
Fixed with #4592
